### ISSUE TITLE
Brainstorm: General Guidelines for Cohort Logos

### ISSUE THREAD

**Issue by Ismael-KG** [open-life-science/branding#34]
This issue is to capture thoughts about establishing guidelines for cohort logos design. A conversation over on Slack has shown that this is a useful thing to think about.

We can discuss what those guidelines might look like through here. The final output would be a guidelines that ensure consistency across icons.

This issue is open for many people to contribute.

**Comment by AdeolaAdekoya** [comment]
Have an understanding of the target audience so the  logo would be something that is of relevance to the cohort

Shouldn't be squashed and in an SVG format to enable it scale without losing quality. 


Adhere to the brand guideline to ensure font and color consistency 

Ensure the logo is legible

**Comment by Lydiaofficial** [comment]
As mentioned on slack, these are some of the guidelines i believe should be considered ;

 1. Cohort name is a determinant factor for the choice of icons.
 2. Colours also play a very important role.
 3. Choice of font
 4. Method of communication
 5. Cohort idea/philosophy. It is important to factor in the sole reason behind the cohort  or the problem the cohort seeks to solve.
 6. And of course, cohort members/community.

**Comment by Bharlikiss** [comment]
I agree with @Lydiaofficial and @AdeolaAdekoya , to add to that we should consider the font choice because fonts have tendencies to be interpreted in different manners, the size is extremely important so as not to lose shape and quality and also if it is legible and works well for both display and printing projects. Then we should consider what logo orientation we are going with  i.e horizontal or stacked  ( to avoid misuse),they tend to make a huge difference. And of course there will be no brand style guide without colors and spacing (between the logo and the surrounding elements)

**Comment by Bisola15** [comment]
Thanks for opening this as an issue. 
As @AdeolaAdekoya has mentioned, the logos should have an svg format for easy scalability. They can also come in png and psd format depending on the use but svg is very important.
The thickness of the stroke in terms of width
The size of the logos and icons. 
Icons come in 24x24, 16x16, 32x32 etc.. but of course having it as an svg means the design of one size is enough. 
The forms of the corners (rounded, sharp or slightly rounded)
Will they be filled or outlined or both (For icons)

For Logos
@Lydiaofficial has said most it. The cohort logo should definitely give us an idea what the cohort is about. Thinking about it, giving the context of what OLS itself about, I can see that the cohort names so far doesn’t have much if anything to do with the they contribute to the community. With that I would say the logos for the cohorts give resonance to their names.

**Comment by latifatakin** [comment]
Intuitiveness: the icons should be easily understood. When users come across the icon, they should be able to interpret and know what it stands for without having to search it online or stress over it mentally. 

Scalability: the icons should be in SVG format to avoid distortion whenever there is a need to increase/decrease the icon size. 

Like @Bisola15  said, they can also come in PNG or PSD but SVG is important. 

The icons should be in tone with the brand identity. The icons should communicate what the brand stands for. 

Style: It is important to decide whether the icons will be filled, outlined or colored.

For Logos, Logo is an important part of brand identity. The logo should be a reflection of the OLS- cohort, when users see the logo, they should be able to relate it to the cohort.

**Comment by Esyteck** [comment]
I totally agree with everyone's suggestion. However I'd like to add a little of my intuition. Firstly, I guess we all have agreed on going ahead with an icon guideline. I'm thinking, just my thinking though that we should streamline our suggested guidelines to make conclusions easier. 

A simple trick to create a memorable logo is to first have a unique concept. I'm sure that's where we are headed. 

We should be looking at keeping these cohorts logos simple and versatile at the same time.  The icon should be able to convey emotions. The logos should depict their names since that is basically what we are looking at.

To be specific, I am thinking we should use rounded corners as one of the guidelines instead of sharp corners. Reason being that rounded corners take less cognitive effort to visualize process than sharp corners. 

Also, I think we should consider filled logo instead of outlined logo. Filled logo are generally easier to recognize without much stress. 

Then for colour, I think, choosing the colour that best suites the cohort names will be ok. Generalizing colours might not be so ideal from my perspective though.

**Comment by AdeolaAdekoya** [comment]
> I totally agree with everyone's suggestion. However I'd like to add a little of my intuition. Firstly, I guess we all have agreed ok going ahead with an icon guideline. I'm thinking, just my thinking though that we should streamline our suggested guidelines to make conclusions easier.
> 
> A simple trick to create a memorable logo is to first have a unique concept. I'm sure that's where we are headed.
> 
> We should be looking at keeping these cohorts logos simple and versatile at the same time. The icon should be able to convey emotions. The logos should depict their names since that is basically what we are looking at.
> 
> To be specific, I thinking we should use rounded corners as one of the guidelines instead of sharp corners. Reason being that rounded corners take less cognitive effort to visualize process than sharp corners.
> 
> Also, I think we should consider filled logo instead of outlined logo. Filled logo are generally easier to recognize without much stress.
> 
> Then for colour, I think, choosing the colour that best suites the cohort names will be ok. Generalizing colours might not be so ideal from my perspective though.

This is a great perspective, especially on the colors part.

**Comment by latifatakin** [comment]
> 


> I totally agree with everyone's suggestion. However I'd like to add a little of my intuition. Firstly, I guess we all have agreed ok going ahead with an icon guideline. I'm thinking, just my thinking though that we should streamline our suggested guidelines to make conclusions easier.
> 
> A simple trick to create a memorable logo is to first have a unique concept. I'm sure that's where we are headed.
> 
> We should be looking at keeping these cohorts logos simple and versatile at the same time. The icon should be able to convey emotions. The logos should depict their names since that is basically what we are looking at.
> 
> To be specific, I thinking we should use rounded corners as one of the guidelines instead of sharp corners. Reason being that rounded corners take less cognitive effort to visualize process than sharp corners.
> 
> Also, I think we should consider filled logo instead of outlined logo. Filled logo are generally easier to recognize without much stress.
> 
> Then for colour, I think, choosing the colour that best suites the cohort names will be ok. Generalizing colours might not be so ideal from my perspective though.


I agree and resonate with your points. Although I have a different opinion on color. 
Each cohort is part of OLS and OLS is a brand with a defined brand identity. 

Color is an important part of brand identity and it should be uniform so as to maintain consistency. 

If each cohort has different colors, then there won’t be consistency.  

I’d suggest that the primary color should be the same for all cohorts then maybe the secondary color can be different. 

What do you think?

**Comment by Esyteck** [comment]
Ok..you are right dear. Absolutely right.

**Comment by caxrii** [comment]
I think the logo of each cohort should be unique.
We should understand the reason for the logo and the color of the logos should be consistent across all cohorts and there should be a design style that would be used when creating the logos and icons.
Accessibility should also be considered when creating the logos and icons for each cohorts.

**Comment by Ismael-KG** [comment]
This is all amazing! I am going to have a go at summing up some of the themes (there is a question at the end, because I think I caused some confusion!):
1. @AdeolaAdekoya reminds us that the logos should be relevant to the cohorts. Remember that this is achieved mostly by cohorts picking their own names (see #15 #16 #17 #18 ). Maybe we can suggest that cohorts should be clear about their "philosophy" (prompted by @Lydiaofficial's thought), which can help with @latifatakin's idea that the logos themselves should be clear and intuitive.
2. @AdeolaAdekoya also mentions file format, with SVG being one standard from which to generate images that scale, and @Bisola15 adds that png and psd files also help
3. Brand guidelines are mentioned and @Lydiaofficial suggests considering colours, font and method of communication, with @Bharlikiss adding logo orientation and and spacing, @Bisola15 mentioning the need to decide about stroke thickness and the roundness of corners, and @latifatakin adding the idea that they be filled, outlined or coloured. @Esyteck's suggestions of filled and rounded logos seem to appeal to fellow contributors.
4. @Caxri7 mentioned accessibility (which I would love if you could expand on!)

With this, we seem to have four themes: _Guidelines to cohorts_, _format_, _brand guidelines_ and _accessibility_.

@yochannah, do you think these can be four separate issues (and this one gets referenced but closed)? Or can we mark somehow that comments are about one of the four themes (for example, I might start a comment with **Accessibility** in bold).

**And the question for everyone!**
I used two words instead of one when opening this issue. Shall we stick to calling the symbols that represent cohorts _logos_ instead of _icons_? (Sorry if I came up with "icons" on my own and just caused havoc!)

**Comment by Bisola15** [comment]
Oh, I think sticking with logos are better @Ismael-KG 😅😅.  As designers, icons and logos are two different elements and the guideline to creating them are different to an extent.

**Comment by caxrii** [comment]
@Ismael-KG @Bisola15 sticking to logos would be great

**Comment by caxrii** [comment]
@Ismael-KG about the accessibility. When designing the logos users with disability should be considered.

**Comment by jilaga** [comment]
@Ismael-KG I actually think icons sound better because they represent the sub-identities of the general OLS brand, bearing in mind that logos can be abstract but icons are clear interpretations as the primary goal is to communicate the names of the OLS cohorts quickly and clearly.

**Comment by jilaga** [comment]
>  I agree @latifatakin  consistency should be maintained in using existing brand colors for the Icons.

**Comment by latifatakin** [comment]
> This is all amazing! I am going to have a go at summing up some of the themes (there is a question at the end, because I think I caused some confusion!):
> 
> 1. @AdeolaAdekoya reminds us that the logos should be relevant to the cohorts. Remember that this is achieved mostly by cohorts picking their own names (see [Design logo for "Masked Cohort", cohort 2 of OLS.  #15](https://github.com/open-life-science/branding/issues/15) [Design a logo for Perseverance, OLS's third cohort #16](https://github.com/open-life-science/branding/issues/16) [Design a logo for Kenaz, OLS's fourth cohort #17](https://github.com/open-life-science/branding/issues/17) [Design a logo for Hope, OLS's fifth cohort #18](https://github.com/open-life-science/branding/issues/18) ). Maybe we can suggest that cohorts should be clear about their "philosophy" (prompted by @Lydiaofficial's thought), which can help with @latifatakin's idea that the logos themselves should be clear and intuitive.
> 2. @AdeolaAdekoya also mentions file format, with SVG being one standard from which to generate images that scale, and @Bisola15 adds that png and psd files also help
> 3. Brand guidelines are mentioned and @Lydiaofficial suggests considering colours, font and method of communication, with @Bharlikiss adding logo orientation and and spacing, @Bisola15 mentioning the need to decide about stroke thickness and the roundness of corners, and @latifatakin adding the idea that they be filled, outlined or coloured. @Esyteck's suggestions of filled and rounded logos seem to appeal to fellow contributors.
> 4. @Caxri7 mentioned accessibility (which I would love if you could expand on!)
> 
> With this, we seem to have four themes: _Guidelines to cohorts_, _format_, _brand guidelines_ and _accessibility_.
> 
> @yochannah, do you think these can be four separate issues (and this one gets referenced but closed)? Or can we mark somehow that comments are about one of the four themes (for example, I might start a comment with **Accessibility** in bold).
> 
> **And the question for everyone!** I used two words instead of one when opening this issue. Shall we stick to calling the symbols that represent cohorts _logos_ instead of _icons_? (Sorry if I came up with "icons" on my own and just caused havoc!)

@Ismael-KG I think sticking to logo will be better. Like @Bisola15 mentioned, logo and icons are quite different and there are different approaches to their design

**Comment by Olakid645** [comment]
> This is all amazing! I am going to have a go at summing up some of the themes (there is a question at the end, because I think I caused some confusion!):
> 
> 1. @AdeolaAdekoya reminds us that the logos should be relevant to the cohorts. Remember that this is achieved mostly by cohorts picking their own names (see [Design logo for "Masked Cohort", cohort 2 of OLS.  #15](https://github.com/open-life-science/branding/issues/15) [Design a logo for Perseverance, OLS's third cohort #16](https://github.com/open-life-science/branding/issues/16) [Design a logo for Kenaz, OLS's fourth cohort #17](https://github.com/open-life-science/branding/issues/17) [Design a logo for Hope, OLS's fifth cohort #18](https://github.com/open-life-science/branding/issues/18) ). Maybe we can suggest that cohorts should be clear about their "philosophy" (prompted by @Lydiaofficial's thought), which can help with @latifatakin's idea that the logos themselves should be clear and intuitive.
> 2. @AdeolaAdekoya also mentions file format, with SVG being one standard from which to generate images that scale, and @Bisola15 adds that png and psd files also help
> 3. Brand guidelines are mentioned and @Lydiaofficial suggests considering colours, font and method of communication, with @Bharlikiss adding logo orientation and and spacing, @Bisola15 mentioning the need to decide about stroke thickness and the roundness of corners, and @latifatakin adding the idea that they be filled, outlined or coloured. @Esyteck's suggestions of filled and rounded logos seem to appeal to fellow contributors.
> 4. @Caxri7 mentioned accessibility (which I would love if you could expand on!)
> 
> With this, we seem to have four themes: _Guidelines to cohorts_, _format_, _brand guidelines_ and _accessibility_.
> 
> @yochannah, do you think these can be four separate issues (and this one gets referenced but closed)? Or can we mark somehow that comments are about one of the four themes (for example, I might start a comment with **Accessibility** in bold).
> 
> **And the question for everyone!** I used two words instead of one when opening this issue. Shall we stick to calling the symbols that represent cohorts _logos_ instead of _icons_? (Sorry if I came up with "icons" on my own and just caused havoc!)

It has been a good comment from everyone that has contributed to the guideline for cohort icons but there seems to be a question that no one is asking or talking about. 

Each Cohorts namely ; (Design logo for "Masked Cohort", cohort 2 of OLS.  #15 https://github.com/open-life-science/branding/issues/16 https://github.com/open-life-science/branding/issues/17 https://github.com/open-life-science/branding/issues/18) should have their own meaning and what they represent differently. 

Starting from at #15 this logo in it's issue should state the purpose it stands to  serve to enable the designers  have an insight of what we are to work with just like the English word sounds "Masked" it  understandable in just a simple English meaning but the purpose it was meant to serve was not stated.

Same as #18 we understand the English word Hope but for us to be able to put this into design we need to know which unit or segment does it stand to represent what is the purpose of that unit so we can create something corresponding and aesthetic.

I will be glad if @Ismael-KG @yochannah can shed more light on this view of mine and update the issues.

Thanks in anticipation for your response.

**Comment by Preciousutos** [comment]
Amazing feedback from everyone. So here's my though on the unique logos for the cohorts. Whatever feel of the logo, be it outlines or filled, should be consistent with the OLS logo. Same goes with the typography and colour. It should be consistent. The cohorts are a sub of OLS and each mean/stand for something unique. @Ismael-KG Icons would have been a better word, but to avoid misconception of the issue, I'll advice we leave it at logo. I also agree with @Olakid645 that there should be some information about each cohorts to better understand the idea behind their naming as this will help in crafting unique logos that resonate with the OLS brand. Each of the cohort logos should have same size (size and shape outlined, including their surrounding environment) and logo saved in svg preferably, for scalability and quality. Colours could be different shades of the brand colour(s).

**Comment by kosonyemedaniella** [comment]
I really do love the feedback from everyone. Here’s my take on this. A design system is important as it ensure uniformity in sense of size , spacing and it also provides clarity for the users. The logo should resonate with the meaning of the cohort  to make it very much easy to interprete. Brand colour should be used as primary colors to set the mood of the brand expression

**Comment by iremide1022** [comment]
I would also suggest that the new logos to be designed should also portray what the entirety of OLS is all about the same colors should be maintained as green symbolizes life🍀( my thoughts)

**Comment by iremide1022** [comment]
@Lydiaofficial @Bisola15 @Preciousutos @AdeolaAdekoya what do we have on ground remember we are to brainstorm together🤗, any conclusions yet?

**Comment by AdeolaAdekoya** [comment]
> This is all amazing! I am going to have a go at summing up some of the themes (there is a question at the end, because I think I caused some confusion!):
> 
> 1. @AdeolaAdekoya reminds us that the logos should be relevant to the cohorts. Remember that this is achieved mostly by cohorts picking their own names (see [Design logo for "Masked Cohort", cohort 2 of OLS.  #15](https://github.com/open-life-science/branding/issues/15) [Design a logo for Perseverance, OLS's third cohort #16](https://github.com/open-life-science/branding/issues/16) [Design a logo for Kenaz, OLS's fourth cohort #17](https://github.com/open-life-science/branding/issues/17) [Design a logo for Hope, OLS's fifth cohort #18](https://github.com/open-life-science/branding/issues/18) ). Maybe we can suggest that cohorts should be clear about their "philosophy" (prompted by @Lydiaofficial's thought), which can help with @latifatakin's idea that the logos themselves should be clear and intuitive.
> 2. @AdeolaAdekoya also mentions file format, with SVG being one standard from which to generate images that scale, and @Bisola15 adds that png and psd files also help
> 3. Brand guidelines are mentioned and @Lydiaofficial suggests considering colours, font and method of communication, with @Bharlikiss adding logo orientation and and spacing, @Bisola15 mentioning the need to decide about stroke thickness and the roundness of corners, and @latifatakin adding the idea that they be filled, outlined or coloured. @Esyteck's suggestions of filled and rounded logos seem to appeal to fellow contributors.
> 4. @Caxri7 mentioned accessibility (which I would love if you could expand on!)
> 
> With this, we seem to have four themes: _Guidelines to cohorts_, _format_, _brand guidelines_ and _accessibility_.
> 
> @yochannah, do you think these can be four separate issues (and this one gets referenced but closed)? Or can we mark somehow that comments are about one of the four themes (for example, I might start a comment with **Accessibility** in bold).
> 
> **And the question for everyone!** I used two words instead of one when opening this issue. Shall we stick to calling the symbols that represent cohorts _logos_ instead of _icons_? (Sorry if I came up with "icons" on my own and just caused havoc!)

Sticking to logo's is great because they mean different things and can cause confusion

**Comment by NPDebs** [comment]
I've just read through this thread, and I must say, we have really talented people here. 💯👏🏼 

Just to add a "few" extra words...
Indeed, a good logo must be scalable, distinct and capable of embodying a brand identity (and message).

OLS encourages diversity and that should reflect in how we create synergy between the various elements of design.

For example, since colors convey different meanings, we can use varying colors for each cohort. 
If that cohort had a unique personality (or value), we can capture it in the color of their logo. (This would require getting information from their mentor(s), possibly through interviews or questionnaires).

Where this is not possible, we can use the already-existing green, which works for just about any brand really.


To ensure visual hierarchy and general graphic balance, choice of typography is extremely important. I think we should maintain the same typography, especially if each cohort has their unique color.

As @Caxri7 mentioned, accessibility takes precedence here.
What's the point in rebranding, if we can't provide the user with a positive, memorable experience?


Finally, optimisation for dark mode! While picking colors, shapes, etc for our brand guide, let's take into account what these would look like in dark mode.


Oh, while we decide, let's also visualise what the logos would look like on future OLS merch too! 😍

In summary, these logos should be minimalistic, unique, memorable, and synergistic.

@Ismael-KG Are you going to open a new issue where we discuss the values of OLS and how we plan to reflect them through the redesign?

**Comment by kryptosjay** [comment]
if am to say i think 
1. knowing  the story behind the organization, i think it will inspire what the designs will look like
2. no matter what never deviate from the color, already its a brand that is already existing taking away the color might not communicate well with clients 
3. consider the font and size you are going to be using let it be clear and readable,
4.  the logo should be scalable, simple and unique

**Comment by Braide1995** [comment]
Very detailed contribution for everyone. I must say, we have so much talents here.

Also, I also think sticking to 'logo' rather than 'icon' is more preferable. 

I think for a successful and effective brand logo, the four cardinal rules to adhere to are;

1. How the logo reflects the company in an honest and unique way
2. Keeping it minimalist and simple 
3. Scalability, like many people mentioned 
4. Artistically balanced logo design in terms of color scheme, size, style, typography and originality
 
Overlooking any of these may impair the final design (My thoughts)

**Comment by Bisola15** [comment]
> @Lydiaofficial @Bisola15 @Preciousutos @AdeolaAdekoya what do we have on ground remember we are to brainstorm together🤗, any conclusions yet?

Hi @iremide1022 Oh so we did not have that meeting anymore. Everything we have has been discussed here since this issue was opened to address that. We put our thoughts here instead

**Comment by MargaretPearl** [comment]
This is a powerful discussion! whats coming out clearly for me is that we need the Brand guidelines that clearly communicate key considerations for future conversations. 
My thoughts and obviously referencing the above contributions is that the logos should clearly convey what each cohort is about but also incorporating the values.  

I think we should also consider the Tone of the content based on the age group we are targeting 

> As mentioned on slack, these are some of the guidelines i believe should be considered ;
> 
> 1. Cohort name is a determinant factor for the choice of icons.
> 2. Colours also play a very important role.
> 3. Choice of font
> 4. Method of communication
> 5. Cohort idea/philosophy. It is important to factor in the sole reason behind the cohort  or the problem the cohort seeks to solve.
> 6. And of course, cohort members/community.
7. Tone of the content

**Comment by stormztee** [comment]
Hello, everyone. I apologize for engaging this so late, @Ismael-KG. I’ve been experiencing serious issues with my PC.

These are some guidelines that I think should be considered:
1. Logos should not be overly detailed to scale properly and work across different platforms.
2. SVG formats of logos should always be provided.
3. Logos should resonate with the cohort name and OLS core values. 
4. The typefaces used should be legible & sized adequately for easy reading. 
5. There should be a preset group of colours to choose from for uniformity across cohort logos. Considerations for people with colour blindness should be made when choosing these colours. (I think this applies to the ‘accessibility’ guideline mentioned by @Caxri7)

**Comment by salmasabo** [comment]
@Ismael-KG heres my thought on the issue.
All logos should look like they're a subsidiary of the ols umbrella.

they should also have consistent colours,shapes,typography and weight.

Logos should also be scalable to bigger or smaller sizes without pixelating 

Each Cohort should have an element that represents what the cohort is about,that way at a glance,it's known that it is under old and what the cohort is about.

**Comment by niyolynate** [comment]
Our cohort  logo should;
Gives a unique identity to OLS.
Help you connect with customers partners.
An impactful logo design distinguishes you from Other Organizations.
Also it should be able to leave an impression of Professionalism.
so they can lay a foundation for all your future branding efforts
by building  brand loyalty.

they should also be 
* Simple
* Original and Distinct
* Versatile
* Targeted
* Memorable
 * Timeless

It gives you a unique identity
Your logo which appears on all your most important marketing tools like your product packaging, stationery, leaflets, business cards, your website, and your social media accounts is your unique mark of ownership. As customers begin to notice your logo everywhere, they begin to associate your logo with your products and services.

 It helps you connect with customers and potential customers
Our cohort logos communicate a lot, the message should be seen in the logo.
Should able to communicate who you are, the value you offer or the type of products or services you offer. But just like many banks do not have dollar signs in their logo design, your logo does not need to be designed explicitly to display what you sell. Instead, it just needs to communicate your brand personality.

**Comment by Preciousutos** [comment]
> Our cohort  logo should;
> Gives a unique identity to OLS.
> Help you connect with customers partners.
> An impactful logo design distinguishes you from Other Organizations.
> Also it should be able to leave an impression of Professionalism.
> so they can lay a foundation for all your future branding efforts
> by building  brand loyalty.
> 
> they should also be 
> * Simple
> * Original and Distinct
> * Versatile
> * Targeted
> * Memorable
>  * Timeless
> 
> It gives you a unique identity
> Your logo which appears on all your most important marketing tools like your product packaging, stationery, leaflets, business cards, your website, and your social media accounts is your unique mark of ownership. As customers begin to notice your logo everywhere, they begin to associate your logo with your products and services.
> 
>  It helps you connect with customers and potential customers
> Our cohort logos communicate a lot, the message should be seen in the logo.
> Should able to communicate who you are, the value you offer or the type of products or services you offer. But just like many banks do not have dollar signs in their logo design, your logo does not need to be designed explicitly to display what you sell. Instead, it just needs to communicate your brand personality.

Nicely said👏. With these I think OLS should really take into consideration, while designing their logo, that it should be timeless and memorable and that's very important, to avoid a change of branding once something new is added. So, I'll suggest an abstract logo.

**Comment by Preciousutos** [comment]
> @Ismael-KG heres my thought on the issue.
> All logos should look like they're a subsidiary of the ols umbrella.
> 
> they should also have consistent colours,shapes,typography and weight.
> 
> Logos should also be scalable to bigger or smaller sizes without pixelating 
> 
> Each Cohort should have an element that represents what the cohort is about,that way at a glance,it's known that it is under old and what the cohort is about.
> 
> 

In addition to this, I will suggest that each cohort logos can have the OLS logo just by the side of the cohort logo. Something like what's on this image
![Screenshot_20221026-091647.jpg](https://user-images.githubusercontent.com/82149094/197972936-3b8b1072-4708-4378-8dd2-82cbd09e4d7d.jpg)

**Comment by Emmzybrown** [comment]
Hello guys, in terms of contributing to this Project sector, how can I record it? I mean, should I post my suggestions here or how is it done please?

**Comment by latifatakin** [comment]
> Hello guys, in terms of contributing to this Project sector, how can I record it? I mean, should I post my suggestions here or how is it done please?

@Emmzybrown yes, you can contribute by commenting on issues. And for some issues, you open a PR. It is better if you ask questions in the slack channel, people tend to reply faster

**Comment by Emmzybrown** [comment]
You all gave wonderful contributions to this. Here are some of the guidelines I formulated , to be considered in creating logos:

1. First we need to consider the our targeted audience, to find their characteristics shared amongst each other. With this understanding, we know the approach to take in creating the logo. 

2. Color also plays an important role. You need to choose a classic and calming color which can also symbolize trustworthiness and maturity. It should be uniformed and consistent. 

3. Size of logo is to be considered. It should be scalable to for better functioning and consistency. 

4. Choosing a font family that is clear and scan-able no matter the size is very important. The font you use, most especially for cohort, should communicate emotions, tone and value.

5. Get logo inspiration from competitors with your targeted audience. When doing this, be sure to clearly set yourself apart from your competitor. 

Here is my contribution to: General Guidelines for Cohort Logos 
@yochannah @Ismael-KG
